CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [v9.2.2] 2021-08-05
```

Comment:
A bit of a nit, but it's good to add something like this to the CHANGELOG for backports:

```
**Please note** changes in 9.2.2 may not yet be released in future versions, as this
is a backport/patch release on the 9.2.x series of releases.  Updates that are
included in the future will have a corresponding CHANGELOG entry in future releases.
```

Reply:
Ah yeah, that seems reasonable. I can add a note like that